### PR TITLE
Merchant index

### PR DESCRIPTION
**Pull request by taphill**
PR for user stories 39-41

  - Allow admins to enable and disable merchant accounts
  - Enabling/disabling a merchant account enables/disables all merchant items
  - Refactored and added tests to Merchant model
  - Added feature tests for functionality
  - Updated merchant FactoryBot factory

  - Resolved merge conflicts with Kate's branch though I did make a few small changes since then, so there may be just a little more work on that front. We'll see once her branch is merged in

### CODE REVIEW

**Review by jdcarey128 — APPROVED** [review]
LGTM!

File: app/views/admin/merchants/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h1 align = "center">Merchants</h1>
```
Comment (katemorris) [inline app/views/admin/merchants/index.html.erb:1]: There should not be spaces around the = in HTML. Don't need to fix now but we should address in refactor.

**Review by katemorris — APPROVED** [review]
Looks good!

**Review by smcunning — APPROVED** [review]
LGTM!

**Pull request merged**